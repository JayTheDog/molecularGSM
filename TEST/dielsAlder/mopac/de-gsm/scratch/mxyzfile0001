PM6 NOSYM 1SCF GRADIENTS UHF AUX 
   MOPAC run 
   ready to go? 
 C -0.825890 1 -1.700451 1 0.637263 1 
 C -1.346599 1 -0.896551 1 -0.321713 1 
 C -1.103508 1 0.506263 1 -0.441169 1 
 C -0.226055 1 1.240589 1 0.369479 1 
 C 1.957618 1 -0.287036 1 0.075082 1 
 C 1.561133 1 0.937010 1 -0.421538 1 
 H -1.987991 1 -1.334270 1 -1.093964 1 
 H -0.205685 1 -1.344826 1 1.444438 1 
 H -0.999607 1 -2.767572 1 0.651865 1 
 H 2.376425 1 -0.401506 1 1.059087 1 
 H 1.817900 1 -1.205336 1 -0.467725 1 
 H 1.282438 1 1.032557 1 -1.468368 1 
 H 1.979063 1 1.855736 1 -0.006190 1 
 H -0.216940 1 2.325390 1 0.272498 1 
 H 0.008507 1 0.902487 1 1.366801 1 
 H -1.590759 1 1.020160 1 -1.270169 1 
