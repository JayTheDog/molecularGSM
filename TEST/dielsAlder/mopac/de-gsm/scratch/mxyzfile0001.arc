

                     SUMMARY OF PM6 CALCULATION, Site No: 14762

                                                       MOPAC2016 (Version: 16.162L)
                                                       Mon Nov 28 15:40:34 2016
                                                       No. of days remaining = 195

           Empirical Formula: C6 H10  =    16 atoms

PM6 NOSYM 1SCF GRADIENTS UHF AUX
   MOPAC run
   ready to go?


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   

          HEAT OF FORMATION       =         64.41266 KCAL/MOL =     269.50255 KJ/MOL
          TOTAL ENERGY            =       -867.94243 EV
          ELECTRONIC ENERGY       =      -3806.88944 EV
          CORE-CORE REPULSION     =       2938.94701 EV
          GRADIENT NORM           =         10.82316
          DIPOLE                  =          0.53152 DEBYE    POINT GROUP:       C1  
          (SZ)                    =          0.000000
          (S**2)                  =          0.881246
          NO. OF ALPHA ELECTRONS  =         17
          NO. OF BETA  ELECTRONS  =         17
          IONIZATION POTENTIAL    =          8.591375 EV
          ALPHA SOMO LUMO (EV)    =         -8.702  0.115
          BETA  SOMO LUMO (EV)    =         -8.591  0.330
          MOLECULAR WEIGHT        =         82.1450
          COSMO AREA              =        139.28 SQUARE ANGSTROMS
          COSMO VOLUME            =        134.49 CUBIC ANGSTROMS

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    13    H     9     5.53900
            H    16    H    10     4.76825
            H    11    H    15     3.00386
          SCF CALCULATIONS        =          1
          WALL-CLOCK TIME         =          0.043 SECONDS
          COMPUTATION TIME        =          0.042 SECONDS


          FINAL GEOMETRY OBTAINED
PM6 NOSYM 1SCF GRADIENTS UHF AUX
   MOPAC run
   ready to go?
  C    -0.82589000 +1  -1.70045100 +1   0.63726300 +1
  C    -1.34659900 +1  -0.89655100 +1  -0.32171300 +1
  C    -1.10350800 +1   0.50626300 +1  -0.44116900 +1
  C    -0.22605500 +1   1.24058900 +1   0.36947900 +1
  C     1.95761800 +1  -0.28703600 +1   0.07508200 +1
  C     1.56113300 +1   0.93701000 +1  -0.42153800 +1
  H    -1.98799100 +1  -1.33427000 +1  -1.09396400 +1
  H    -0.20568500 +1  -1.34482600 +1   1.44443800 +1
  H    -0.99960700 +1  -2.76757200 +1   0.65186500 +1
  H     2.37642500 +1  -0.40150600 +1   1.05908700 +1
  H     1.81790000 +1  -1.20533600 +1  -0.46772500 +1
  H     1.28243800 +1   1.03255700 +1  -1.46836800 +1
  H     1.97906300 +1   1.85573600 +1  -0.00619000 +1
  H    -0.21694000 +1   2.32539000 +1   0.27249800 +1
  H     0.00850700 +1   0.90248700 +1   1.36680100 +1
  H    -1.59075900 +1   1.02016000 +1  -1.27016900 +1
 
