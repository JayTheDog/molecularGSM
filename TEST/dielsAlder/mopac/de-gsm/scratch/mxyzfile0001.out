 *******************************************************************************
 ** Site#: 14762        For non-commercial use only     Version 16.162L 64BITS**
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 16.162L, James J. P. Stewart,   **
 ** Stewart Computational Chemistry, web: HTTP://OpenMOPAC.net. Days left: 195**
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM6 CALCULATION RESULTS


 *******************************************************************************
 *  CALCULATION DONE:                                Mon Nov 28 15:40:34 2016  *
 *  PM6        - The PM6 Hamiltonian to be used
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  NOSYM      - POINT-GROUP SYMMETRY SET TO C1
 *  1SCF       - DO 1 SCF AND THEN STOP 
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY  7200.000 SECONDS
 *  AUX        - OUTPUT AUXILIARY INFORMATION
 *  GRADIENTS- ALL GRADIENTS TO BE PRINTED
 *******************************************************************************
PM6 NOSYM 1SCF GRADIENTS UHF AUX
   MOPAC run
   ready to go?
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C         -0.82589000  *  -1.70045100  *   0.63726300  *
     2       C         -1.34659900  *  -0.89655100  *  -0.32171300  *
     3       C         -1.10350800  *   0.50626300  *  -0.44116900  *
     4       C         -0.22605500  *   1.24058900  *   0.36947900  *
     5       C          1.95761800  *  -0.28703600  *   0.07508200  *
     6       C          1.56113300  *   0.93701000  *  -0.42153800  *
     7       H         -1.98799100  *  -1.33427000  *  -1.09396400  *
     8       H         -0.20568500  *  -1.34482600  *   1.44443800  *
     9       H         -0.99960700  *  -2.76757200  *   0.65186500  *
    10       H          2.37642500  *  -0.40150600  *   1.05908700  *
    11       H          1.81790000  *  -1.20533600  *  -0.46772500  *
    12       H          1.28243800  *   1.03255700  *  -1.46836800  *
    13       H          1.97906300  *   1.85573600  *  -0.00619000  *
    14       H         -0.21694000  *   2.32539000  *   0.27249800  *
    15       H          0.00850700  *   0.90248700  *   1.36680100  *
    16       H         -1.59075900  *   1.02016000  *  -1.27016900  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM         X         Y         Z

     1         C       -0.8259   -1.7005    0.6373
     2         C       -1.3466   -0.8966   -0.3217
     3         C       -1.1035    0.5063   -0.4412
     4         C       -0.2261    1.2406    0.3695
     5         C        1.9576   -0.2870    0.0751
     6         C        1.5611    0.9370   -0.4215
     7         H       -1.9880   -1.3343   -1.0940
     8         H       -0.2057   -1.3448    1.4444
     9         H       -0.9996   -2.7676    0.6519
    10         H        2.3764   -0.4015    1.0591
    11         H        1.8179   -1.2053   -0.4677
    12         H        1.2824    1.0326   -1.4684
    13         H        1.9791    1.8557   -0.0062
    14         H       -0.2169    2.3254    0.2725
    15         H        0.0085    0.9025    1.3668
    16         H       -1.5908    1.0202   -1.2702

 General Reference for PM6:
 "Optimization of Parameters for Semiempirical Methods V: Modification of NDDO Approximations
 and Application to 70 Elements", J. J. P. Stewart, J. Mol. Mod., 13, 1173-1213 (2007)
 URL: http://www.springerlink.com/content/ar33482301010477/fulltext.pdf

           Empirical Formula: C6 H10  =    16 atoms



      MOLECULAR POINT GROUP   :   C1  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =   17
                           NO. OF BETA  ELECTRONS =   17

 -------------------------------------------------------------------------------
PM6 NOSYM 1SCF GRADIENTS UHF AUX
   MOPAC run
   ready to go?


     1SCF WAS SPECIFIED, SO BFGS WAS NOT USED                 
     SCF FIELD WAS ACHIEVED                                   


                              PM6 CALCULATION
                                                       MOPAC2016 (Version: 16.162L)
                                                       Mon Nov 28 15:40:34 2016
                                                       No. of days remaining = 195




          FINAL HEAT OF FORMATION =         64.41266 KCAL/MOL =     269.50255 KJ/MOL


          TOTAL ENERGY            =       -867.94243 EV
          ELECTRONIC ENERGY       =      -3806.88944 EV  POINT GROUP:     C1  
          CORE-CORE REPULSION     =       2938.94701 EV
          COSMO AREA              =        139.28 SQUARE ANGSTROMS
          COSMO VOLUME            =        134.49 CUBIC ANGSTROMS

          GRADIENT NORM           =         10.82316
          IONIZATION POTENTIAL    =          8.591375 EV
          ALPHA SOMO LUMO (EV)    =         -8.702  0.115
          BETA  SOMO LUMO (EV)    =         -8.591  0.330
          NO. OF ALPHA ELECTRONS  =         17
          NO. OF BETA  ELECTRONS  =         17
          MOLECULAR WEIGHT        =         82.1450

          MOLECULAR DIMENSIONS (Angstroms)

            Atom       Atom       Distance
            H    13    H     9     5.53900
            H    16    H    10     4.76825
            H    11    H    15     3.00386


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.043 SECONDS
          COMPUTATION TIME        =          0.039 SECONDS



       FINAL  POINT  AND  DERIVATIVES

   PARAMETER     ATOM    TYPE            VALUE       GRADIENT
      1          1  C    CARTESIAN X    -0.825890    -0.224341  KCAL/ANGSTROM
      2          1  C    CARTESIAN Y    -1.700451     3.593457  KCAL/ANGSTROM
      3          1  C    CARTESIAN Z     0.637263    -0.996251  KCAL/ANGSTROM
      4          2  C    CARTESIAN X    -1.346599     0.500097  KCAL/ANGSTROM
      5          2  C    CARTESIAN Y    -0.896551     0.576486  KCAL/ANGSTROM
      6          2  C    CARTESIAN Z    -0.321713     0.957590  KCAL/ANGSTROM
      7          3  C    CARTESIAN X    -1.103508     0.180931  KCAL/ANGSTROM
      8          3  C    CARTESIAN Y     0.506263    -2.067879  KCAL/ANGSTROM
      9          3  C    CARTESIAN Z    -0.441169     0.856143  KCAL/ANGSTROM
     10          4  C    CARTESIAN X    -0.226055     1.838664  KCAL/ANGSTROM
     11          4  C    CARTESIAN Y     1.240589    -2.774617  KCAL/ANGSTROM
     12          4  C    CARTESIAN Z     0.369479     3.497447  KCAL/ANGSTROM
     13          5  C    CARTESIAN X     1.957618    -1.832713  KCAL/ANGSTROM
     14          5  C    CARTESIAN Y    -0.287036     2.678980  KCAL/ANGSTROM
     15          5  C    CARTESIAN Z     0.075082    -1.779099  KCAL/ANGSTROM
     16          6  C    CARTESIAN X     1.561133    -0.322624  KCAL/ANGSTROM
     17          6  C    CARTESIAN Y     0.937010    -5.135562  KCAL/ANGSTROM
     18          6  C    CARTESIAN Z    -0.421538     0.988203  KCAL/ANGSTROM
     19          7  H    CARTESIAN X    -1.987991     0.573921  KCAL/ANGSTROM
     20          7  H    CARTESIAN Y    -1.334270    -0.098704  KCAL/ANGSTROM
     21          7  H    CARTESIAN Z    -1.093964    -0.076609  KCAL/ANGSTROM
     22          8  H    CARTESIAN X    -0.205685    -0.099727  KCAL/ANGSTROM
     23          8  H    CARTESIAN Y    -1.344826    -0.051594  KCAL/ANGSTROM
     24          8  H    CARTESIAN Z     1.444438     0.328547  KCAL/ANGSTROM
     25          9  H    CARTESIAN X    -0.999607     0.002265  KCAL/ANGSTROM
     26          9  H    CARTESIAN Y    -2.767572    -1.941295  KCAL/ANGSTROM
     27          9  H    CARTESIAN Z     0.651865    -0.894183  KCAL/ANGSTROM
     28         10  H    CARTESIAN X     2.376425    -1.091070  KCAL/ANGSTROM
     29         10  H    CARTESIAN Y    -0.401506     0.253459  KCAL/ANGSTROM
     30         10  H    CARTESIAN Z     1.059087     0.596993  KCAL/ANGSTROM
     31         11  H    CARTESIAN X     1.817900     1.321333  KCAL/ANGSTROM
     32         11  H    CARTESIAN Y    -1.205336     0.171889  KCAL/ANGSTROM
     33         11  H    CARTESIAN Z    -0.467725    -0.403042  KCAL/ANGSTROM
     34         12  H    CARTESIAN X     1.282438    -0.172712  KCAL/ANGSTROM
     35         12  H    CARTESIAN Y     1.032557    -0.205634  KCAL/ANGSTROM
     36         12  H    CARTESIAN Z    -1.468368     0.332749  KCAL/ANGSTROM
     37         13  H    CARTESIAN X     1.979063     1.142243  KCAL/ANGSTROM
     38         13  H    CARTESIAN Y     1.855736     1.431498  KCAL/ANGSTROM
     39         13  H    CARTESIAN Z    -0.006190     0.132373  KCAL/ANGSTROM
     40         14  H    CARTESIAN X    -0.216940     0.089626  KCAL/ANGSTROM
     41         14  H    CARTESIAN Y     2.325390     1.223003  KCAL/ANGSTROM
     42         14  H    CARTESIAN Z     0.272498    -0.605208  KCAL/ANGSTROM
     43         15  H    CARTESIAN X     0.008507    -1.145548  KCAL/ANGSTROM
     44         15  H    CARTESIAN Y     0.902487     1.879763  KCAL/ANGSTROM
     45         15  H    CARTESIAN Z     1.366801    -3.495030  KCAL/ANGSTROM
     46         16  H    CARTESIAN X    -1.590759    -0.760345  KCAL/ANGSTROM
     47         16  H    CARTESIAN Y     1.020160     0.466749  KCAL/ANGSTROM
     48         16  H    CARTESIAN Z    -1.270169     0.559377  KCAL/ANGSTROM




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1       C         -0.82589000  *  -1.70045100  *   0.63726300  *
     2       C         -1.34659900  *  -0.89655100  *  -0.32171300  *
     3       C         -1.10350800  *   0.50626300  *  -0.44116900  *
     4       C         -0.22605500  *   1.24058900  *   0.36947900  *
     5       C          1.95761800  *  -0.28703600  *   0.07508200  *
     6       C          1.56113300  *   0.93701000  *  -0.42153800  *
     7       H         -1.98799100  *  -1.33427000  *  -1.09396400  *
     8       H         -0.20568500  *  -1.34482600  *   1.44443800  *
     9       H         -0.99960700  *  -2.76757200  *   0.65186500  *
    10       H          2.37642500  *  -0.40150600  *   1.05908700  *
    11       H          1.81790000  *  -1.20533600  *  -0.46772500  *
    12       H          1.28243800  *   1.03255700  *  -1.46836800  *
    13       H          1.97906300  *   1.85573600  *  -0.00619000  *
    14       H         -0.21694000  *   2.32539000  *   0.27249800  *
    15       H          0.00850700  *   0.90248700  *   1.36680100  *
    16       H         -1.59075900  *   1.02016000  *  -1.27016900  *

                             CARTESIAN COORDINATES

   1    C       -0.825890000    -1.700451000     0.637263000
   2    C       -1.346599000    -0.896551000    -0.321713000
   3    C       -1.103508000     0.506263000    -0.441169000
   4    C       -0.226055000     1.240589000     0.369479000
   5    C        1.957618000    -0.287036000     0.075082000
   6    C        1.561133000     0.937010000    -0.421538000
   7    H       -1.987991000    -1.334270000    -1.093964000
   8    H       -0.205685000    -1.344826000     1.444438000
   9    H       -0.999607000    -2.767572000     0.651865000
  10    H        2.376425000    -0.401506000     1.059087000
  11    H        1.817900000    -1.205336000    -0.467725000
  12    H        1.282438000     1.032557000    -1.468368000
  13    H        1.979063000     1.855736000    -0.006190000
  14    H       -0.216940000     2.325390000     0.272498000
  15    H        0.008507000     0.902487000     1.366801000
  16    H       -1.590759000     1.020160000    -1.270169000


           Empirical Formula: C6 H10  =    16 atoms



      MOLECULAR POINT GROUP   :   C1  


          ALPHA EIGENVALUES  
 -28.27070 -26.52793 -24.87599 -22.00932 -20.48478 -18.22156 -16.67449 -15.84820
 -14.58661 -14.41418 -13.56498 -12.48740 -12.15132 -12.07772 -11.87526 -10.45422
  -8.70224   0.11500   1.60164   2.58940   4.88661   5.16657   5.57217   5.63581
   5.73894   5.91198   6.02970   6.21130   6.29183   6.45807   6.53262   6.57882
   6.61227   6.79501


           BETA EIGENVALUES 
 -28.28685 -26.50981 -24.87914 -22.05047 -20.40783 -18.25998 -16.69583 -15.82194
 -14.58924 -14.37491 -13.55308 -12.53484 -12.14420 -12.12799 -11.87579 -10.54567
  -8.59137   0.33047   1.50892   2.51846   4.82711   5.13102   5.61948   5.65362
   5.68485   5.92411   6.02041   6.22174   6.31433   6.39965   6.47264   6.64559
   6.73948   6.74716


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       
    1          C          -0.328815        4.3288     1.12421     3.20461
    2          C          -0.107274        4.1073     1.09963     3.00765
    3          C          -0.166796        4.1668     1.10997     3.05682
    4          C          -0.259336        4.2593     1.12459     3.13474
    5          C          -0.323637        4.3236     1.12635     3.19729
    6          C          -0.230014        4.2300     1.11821     3.11181
    7          H           0.134289        0.8657     0.86571
    8          H           0.151184        0.8488     0.84882
    9          H           0.143640        0.8564     0.85636
   10          H           0.138714        0.8613     0.86129
   11          H           0.147740        0.8523     0.85226
   12          H           0.141227        0.8588     0.85877
   13          H           0.137771        0.8622     0.86223
   14          H           0.137286        0.8627     0.86271
   15          H           0.148037        0.8520     0.85196
   16          H           0.135984        0.8640     0.86402
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.152     0.508    -0.110     0.541
 HYBRID         0.020    -0.007     0.062     0.065
 SUM            0.172     0.501    -0.048     0.532


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz   
    1  C   1.12421   1.05105   1.09417   1.05939
    2  C   1.09963   1.00869   0.97971   1.01925
    3  C   1.10997   1.00549   0.99452   1.05682
    4  C   1.12459   0.97305   1.08742   1.07428
    5  C   1.12635   1.03546   1.05420   1.10763
    6  C   1.11821   0.97480   1.04262   1.09439
    7  H   0.86571
    8  H   0.84882
    9  H   0.85636
   10  H   0.86129
   11  H   0.85226
   12  H   0.85877
   13  H   0.86223
   14  H   0.86271
   15  H   0.85196
   16  H   0.86402


                    (SZ)    =    0.000000
                    (S**2)  =    0.881246


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   
         Atomic
    1  C -0.5235 -0.014 -0.318 -0.030 -0.161
    2  C  0.3885  0.015  0.213  0.040  0.121
    3  C -0.6642 -0.021 -0.361 -0.057 -0.226
    4  C  0.5239  0.031  0.340  0.045  0.107
    5  C  0.7276  0.019  0.548  0.038  0.123
    6  C -0.4472 -0.024 -0.318 -0.029 -0.076
    7  H -0.0139 -0.014
    8  H  0.0174  0.017
    9  H  0.0182  0.018
   10  H -0.0242 -0.024
   11  H -0.0242 -0.024
   12  H  0.0185  0.018
   13  H  0.0222  0.022
   14  H -0.0191 -0.019
   15  H -0.0234 -0.023
   16  H  0.0232  0.023

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.07 SECONDS

 == MOPAC DONE ==
